.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# Pull Request Checklist
+
+* [ ] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
+* [ ] Have you signed the [Typesafe CLA](https://www.typesafe.com/contribute/cla)?
+* [ ] Have you [squashed your commits](https://www.playframework.com/documentation/2.4.x/WorkingWithGit#Squashing-commits)?
```

Comment:
Just noticed this was still on 2.4.x. We could switch to "latest": https://www.playframework.com/documentation/latest/WorkingWithGit#Squashing-commits